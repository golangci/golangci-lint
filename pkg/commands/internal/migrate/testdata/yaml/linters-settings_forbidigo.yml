issues:
  # Only to not generate unrelated elements inside golden.
  exclude-use-default: false
  # Only to not generate unrelated elements inside golden.
  exclude-generated: strict
  # Only to not generate unrelated elements inside golden.
  exclude-dirs-use-default: false

linters-settings:
  forbidigo:
    forbid:
      - ^print(ln)?$
      - p: ^fmt\.Print.*$
        msg: Do not commit print statements.
      - 'fmt\.Print.*(# Do not commit print statements\.)?'
      - ^spew\.(ConfigState\.)?Dump$
      - p: ^v1.Dump$
        pkg: ^example.com/pkg/api/v1$
    exclude-godoc-examples: false
    analyze-types: true
