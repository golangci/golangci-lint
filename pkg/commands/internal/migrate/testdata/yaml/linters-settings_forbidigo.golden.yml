version: "2"
linters:
  settings:
    forbidigo:
      forbid:
        - pattern: ^print(ln)?$
        - pattern: ^fmt\.Print.*$
          msg: Do not commit print statements.
        - pattern: fmt\.Print.*(# Do not commit print statements\.)?
        - pattern: ^spew\.(ConfigState\.)?Dump$
        - pattern: ^v1.Dump$
          pkg: ^example.com/pkg/api/v1$
        - pkg: ^github.com/howeyc/gopass$
          msg: github.com/howeyc/gopass is archived, use golang.org/x/term instead
      exclude-godoc-examples: false
      analyze-types: true
